application/templates/prometheus-rule.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.prometheusRule -}}
+{{- if .Values.prometheusRule.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+{{- if .Values.prometheusRule.labels -}}
+  labels:
+{{ toYaml .Values.prometheusRule.labels | indent 4 }}
```

Comment:
can we also please default labels of helm here as other resources? Is there any downside of that?

Reply:
done